packages/react/src/wallet/useWalletLogin.ts

```diff
@@ -3,13 +3,14 @@ import {
   UserRejectedError,
   WalletConnectionError,
 } from '@lens-protocol/domain/entities';
+import { WalletLoginResult } from '@lens-protocol/domain/use-cases/wallets';
 import { Signer } from 'ethers';
 
 import { Operation, useOperation } from '../helpers/operations';
 import { useWalletLoginController } from './adapters/useWalletLoginController';
 
 export type WalletLoginOperation = Operation<
-  void,
+  WalletLoginResult,
```

Comment:
This one changes the return type of user-facing hook, do I need to update any docs?

Reply:
Reference documentation can be regenerated once this is released on a stable version. Manual documentation wise we could start adding some details to [this guide](https://docs.lens.xyz/docs/hooks-authentication) once this is merged and released under `next` tag.